src/Uno.UWP/System/KeyEventArgs.cs

```diff
@@ -22,5 +23,7 @@ internal KeyEventArgs(string deviceId, VirtualKey virtualKey, CorePhysicalKeySta
 		public VirtualKey VirtualKey { get; }
 
 		public string DeviceId { get; }
+
+		public VirtualKeyModifiers KeyboardModifiers { get; }
```

Comment:
The `KeyEventArgs` is a public class, so this property has to be `internal`:
```suggestion
		internal VirtualKeyModifiers KeyboardModifiers { get; }
```